common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -7303,6 +7304,12 @@ public Map<String, String> subtree(String string) {
     return ret;
   }
 
+  public static boolean shouldComputeLineage(HiveConf conf) {
+    Collection<String> lineage_filter =
```

Comment:
Please change the name of `lineage_filter` to `lineageFilter`

Reply:
Ack